types/node-insights/index.d.ts

```diff
@@ -0,0 +1,85 @@
+// Type definitions for node-insights 1.1
+// Project: https://github.com/flightstats/node-insights#readme
+// Definitions by: Walter Rumsby <https://github.com/wrumsby>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+export interface Config {
+    insertKey: string;
+    queryKey: string;
+    accountId: string;
+    timerInterval?: number;
+    maxPending?: number;
+    defaultEventType?: string;
+    enabled?: boolean;
+    gzip?: boolean;
+}
+
+export type CallbackFunc = (err: Error | null | undefined, responseBody: string) => void;
+
+export default class Insights {
```

Comment:
As @jloveridge has mentioend, the `default` should be removed if there's no default export.